clap.zig

```diff
@@ -92,6 +92,7 @@ pub fn Param(comptime Id: type) type {
         id: Id,
         names: Names = Names{},
         takes_value: Values = .none,
+        required: bool = false,
```

Comment:
Ofc, the question then becomes. How is `many` parsed? Is it `--many <v>...` or `--many [v]...`? I would argue for `<v>...` as each value is not optional, but there can be `0..N` of them

Reply:
For me `[v]...` means that I can omit the flag. `<v>...` indicates that I have to add the flag with at least one value. This could be even enhanced later to at least `k` values, but that does not seem a severe requirement to me.
If I want only the flag without value there is `<bool>`.

Reply:
> For me [v]... means that I can omit the flag. <v>... indicates that I have to add the flag with at least one value.

Hmm. Alright. I think we have to look out and see what the standard is here and pick that :) I'll try to have a look around later.